lib/pavilion/plugins/commands/version.py

```diff
@@ -0,0 +1,40 @@
+"""Displays the Pavilion version."""
+
+import errno
+import re
+
+from pavilion import commands
+from pavilion import output
+from pavilion.output import fprint
+
+
+class VersionCommand(commands.Command):
+
+    def __init__(self):
+        super().__init__(
+            name='version',
+            description='Display version of Pavilion.',
+            short_help="Displays the current version of Pavilion.",
+        )
+
+    def run(self, pav_cfg, args):
+        """Run fetches the version number from RELEASE.txt"""
+
+        version_path = str(pav_cfg.pav_root) + '/RELEASE.txt'
```

Comment:
import pathlib
...
`version_path = pathlib.Path(pav_cfg.pav_root) / 'RELEASE.txt'`

Reply:
pav_cfg.pav_root is already a pathlib object, so:
```
version_path = pav_cfg.pav_root / 'RELEASE.txt'
```